src/app/services/auth/auth.service.ts

```diff
@@ -3,23 +3,32 @@ import { AngularFireAuth } from "angularfire2/auth";
 import { Observable } from 'rxjs';
 import * as firebase from 'firebase/app';
 
-//import { Router, NavigationExtras, ActivatedRoute } from '@angular/router';
-//import { FirebaseService } from 'src/app/services/firebase/firebase.service';
-
 @Injectable({
   providedIn: 'root'
 })
 export class AuthService {
-  user: Observable<firebase.User>;
```

Comment:
So did you remove 'user' as a property altogether, or did you replace it with another?